spec/rest.yaml

```diff
@@ -363,6 +363,22 @@ components:
           additionalProperties:
             type: string
 
+    CreateNamespaceResponse:
```

Comment:
Instead of making Create return Get response, we now have a dedicated create response. This is inspired by recent discussions in IRC that Create might need to return a transaction handle and the response would be different from a Get. Similarly, Alter and Drop should also have dedicated responses.